mat/lu.go

```diff
@@ -177,6 +177,11 @@ func (lu *LU) Pivot(dst []int) []int {
 	return dst
 }
 
+// Pivot is deprecated, use RowPivots instead.
```

Comment:
There is a magic incantation to get the on-line docs to hid deprecated labels.

```suggestion
// Deprecated: Use RowPivots instead.
```

Reply:
Thanks, done.